content/schedule/coach_meeting_bapc.md

```diff
@@ -5,10 +5,12 @@ day: sunday
 start: 12:15
 duration: 1h15m
 width: 1
-location: Aula - TBD
+location: Aula - Hall A
 people: coaches from Benelux area
 ---
 ## Location
-The BAPC coach meeting takes place in the Auditorium of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+The BAPC coach meeting takes place in the Hall A of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+You will be guided to the hall by volunteers from the Auditorium.
+Please use the elevator to reach the auditorium from the ground floor.
```

Comment:
```suggestion
Please use the elevator to reach the top entrance of Hall A from the ground floor.
```

And same for all other coach meetings :slightly_smiling_face:

Reply:
This is intentional, If you take the elevator to Hall A/B, you end up in the foyer and that is where we don't want the coaches on Sunday.

Reply:
Check, makes sense, thanks! :slightly_smiling_face:

Reply:
Unless i'm greatly mistaken, but there is no elevator on the backside of the aula.